ops/cloudbuild.ci.yaml

```diff
@@ -20,3 +30,13 @@ availableSecrets:
       versionName: projects/$PROJECT_ID/secrets/VITE_CLOUDINARY_SECRET/versions/1
     - env: 'VITE_BIBLE_API_KEY'
       versionName: projects/$PROJECT_ID/secrets/VITE_BIBLE_API_KEY/versions/1
+    - env: 'VITE_S3_ACCESS_KEY_ID'
```

Comment:
The `VITE_S3_ACCESS_KEY_ID` and `VITE_S3_SECRET_ACCESS_KEY` should remain in secrets. We can't avoid that.